modules/selenium/src/test/java/org/testcontainers/junit/BrowserWebDriverContainerTest.java

```diff
@@ -39,4 +42,33 @@ public void provideDefaultNoProxyEnvironmentIfNotSet() {
             assertEquals("no_proxy should be set to default if not already present", "localhost", noProxy);
         }
     }
+
+
+    @Test
+    public void createContainerWithShmVolume() {
+        try (
+            BrowserWebDriverContainer webDriverContainer = new BrowserWebDriverContainer()
+        ) {
+            webDriverContainer.start();
+
+            assertEquals("Shm mounts present", webDriverContainer.getContainerInfo().getMounts().size(), 1);
+            final InspectContainerResponse.Mount shmMount = webDriverContainer.getContainerInfo().getMounts().get(0);
+            assertEquals("Shm mount source is correct", "/dev/shm", shmMount.getSource());
+            assertEquals("Shm mount destination is correct", "/dev/shm", shmMount.getDestination().getPath());
+            assertEquals("Shm mount mode is correct", shmMount.getMode(), "rw");
+        }
+    }
+
+    @Test
+    public void createContainerWithoutShmVolume() {
+        try (
+            BrowserWebDriverContainer webDriverContainer = (BrowserWebDriverContainer) new BrowserWebDriverContainer()
+             .withSharedMemorySize(512 * FileUtils.ONE_MB)
+        ) {
+            webDriverContainer.start();
+            assertEquals("Shared memory size is configured", 512 * FileUtils.ONE_MB, webDriverContainer.getShmSize());
+            assertEquals("No mounts present", webDriverContainer.getContainerInfo().getMounts().size(), 0);
```

Comment:
please check that there are no `shm` mounts, not just any, so that if we add some volumes in future, it won't break the test (same in `createContainerWithShmVolume`)

Reply:
FTR this one was only partially resolved. @glefloch please do not resolve other's comments, let the author resolve it when he thinks it is so

Reply:
Sorry @bsideup, I will open another pull request with this fix. I thought it was only for this test...

Reply:
no worries, @rnorth is working on it in #1948